[BUG] python gh-workflow fails blocking all builds

**Describe the bug**

```
Run cd user_tools && tox -e pylint,flake8,python3.8 -- tests/test_diagnostic.py
ROOT: tox-gh-actions won't override envlist because envlist is explicitly given via TOXENV or -e option
ROOT: HandledError| provided environments not found in configuration file:
python3.8
Error: Process completed with exit code 254.
```

--- a/.github/workflows/python-unit-test.yml
+++ b/.github/workflows/python-unit-test.yml
@@ -22,20 +22,23 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
 
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.8'
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
 
-    - name: Install tox
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install tox
-        python -m pip install --pre tox-gh-actions
+      - name: Install tox
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install tox
+          python -m pip install --pre tox-gh-actions
 
-    - name: Run tox test
-      run: cd user_tools && tox -e pylint,flake8,python3.8 -- tests/test_diagnostic.py
+      - name: Run tox test
+        run: cd user_tools && tox

--- a/user_tools/tox.ini
+++ b/user_tools/tox.ini
@@ -11,6 +11,12 @@ envlist =
     flake8
 isolated_build = True
 
+[gh-actions]
+python =
+    3.8: python3.8, pylint, flake8
+    3.9: python3.9, pylint, flake8
+    3.10: python3.10, pylint, flake8
+
 [testenv]
 deps =
     pytest-cov

